app/assets/javascripts/browsered/worldpay-3ds-flex-ddc.js

```diff
@@ -54,7 +59,12 @@ const submitWithWorldpay3dsFlexDdcResult = form => {
       const { MessageType, SessionId, Status } = JSON.parse(event.data)
       if (MessageType === 'profile.completed') {
         if (Status === true && typeof SessionId === 'string') {
+          updateDdcStatusField('valid DDC result')
           addWorldpaySessionIdToForm(form, SessionId)
+        } else if (!Status) {
+          updateDdcStatusField('DDC response had Status of false')
```

Comment:
I suppose technically it could be that it’s missing or an empty string or something.